ansible/DockerInventory.json

```diff
@@ -328,6 +344,6 @@
                 "port": "32790"
             }
         ],
-        "containersCount": 4
+        "containersCount": 3
     }
 ]
```

Comment:
There should perhaps be a newline character on the end of this, but I'll approve regardless :-)

Reply:
The json file is generated by the https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updateDockerStaticInventory.py script, so I can add a change in there to create a new line at the end of the file